soc/xtensa/intel_adsp/tools/cavstool.py

```diff
@@ -24,9 +25,17 @@
 PORT_LOG = 9999
 PORT_REQ = PORT_LOG + 1
 BUF_SIZE = 4096
+
+# Define the command and the max size
 CMD_LOG_START = "start_log"
-CMD_LOG_STOP = "stop_log"
 CMD_DOWNLOAD = "download"
+CMD_SZ = 16
```

Comment:
```suggestion
MAX_CMD_SZ = 16
```
Otherwise it looks like commands have constant size (which would be more correct, see below).

But don't do it now to save time. Change it only if another reviewer finds a serious issue and requests other changes.

Reply:
I got it, I was thinking to make the command length to be 16(with '\0') , but maybe we should change it in a higher level, that's way I did not make it fit totally.

Reply:
Got it fixed, thanks!